{
  "errorMessage": "Object of type Decimal is not JSON serializable",
  "errorType": "TypeError",
  "requestId": "7137359b-ef18-4aa3-a001-4ad21935d743",
  "stackTrace": [
    "  File \"/var/task/lambda_function.py\", line 309, in lambda_handler\n    fallback_response = invoke_fallback({ \"body\": json.dumps(fallback_body) })\n",
    "  File \"/var/lang/lib/python3.13/json/__init__.py\", line 231, in dumps\n    return _default_encoder.encode(obj)\n",
    "  File \"/var/lang/lib/python3.13/json/encoder.py\", line 200, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n",
    "  File \"/var/lang/lib/python3.13/json/encoder.py\", line 261, in iterencode\n    return _iterencode(o, 0)\n",
    "  File \"/var/lang/lib/python3.13/json/encoder.py\", line 180, in default\n    raise TypeError(f'Object of type {o.__class__.__name__} '\n"
  ]
}
